precompiles/auctions/events.go

```diff
@@ -0,0 +1,96 @@
+// Copyright Tharsis Labs Ltd.(Evmos)
+// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/evmos/blob/main/LICENSE)
+
+package auctions
+
+import (
+	"math/big"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/common"
+	ethtypes "github.com/ethereum/go-ethereum/core/types"
+	contractutils "github.com/evmos/evmos/v19/contracts/utils"
+	"github.com/evmos/evmos/v19/x/evm/core/vm"
+)
+
+const (
+	// EventTypeBid defines the event type for the auctions Bid transaction.
```

Comment:
Nit: we can add in this file a comment about the AuctionEnd event, that is emitted during the hook. Just for reference